### PR TITLE
Cover the EE eval; update the instructions

### PR DESCRIPTION
**Pull request by rbotzer**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -52,9 +52,8 @@ COPY entrypoint.sh /entrypoint.sh
 #   3000 – service port, for client connections
 #   3001 – fabric port, for cluster communication
 #   3002 – mesh port, for cluster heartbeat
-#   3003 – info port
 #
-EXPOSE 3000 3001 3002 3003
+EXPOSE 3000 3001 3002
```
Comment (rbotzer) [inline Dockerfile:56]: @volmarl I want to remove 3003, so I did it here and in the examples. Anything else?

Comment (volmarl) [reply Dockerfile:56]: That should be it  in terms of expose ports.

File: README.md [hunk 2]
```diff
@@ -1,122 +1,145 @@
...
-## Block Storage
+### Block Storage
 
-Docker run offers the ability to expose hosts block devices to a running container. The --device option can be used to map a host block device within a container.
+Docker run offers the ability to expose hosts block devices to a running container. The `--device` option can be used to map a host block device within a container.
 
 Example mapping /dev/sdc to /dev/xvdc on a running container.
 
+```sh
+docker run -tid --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 -v /aerospike-server-enterprise.docker:/etc/aerospike/ --device '/dev/sdc:/dev/xvdc' -e "FEATURE_KEY_FILE=/etc/aerospike/features.conf" aerospike-server-enterprise:latest
```
Comment (rbotzer) [inline README.md:138]: @volmarl do you know what this /aerospike-server-enterprise.docker path in the -v mapping is about? It's confusing to me, but I wasn't sure what to replace it with

File: README.md [hunk 3]
```diff
@@ -137,7 +160,26 @@ A full example:
 
 ## Clustering
 
-Aerospike recommends using multicast clustering whenever possible, however, we are currently working to figure out how to best support multicast via Docker. For the time being, it will be best to setup Mesh Clustering. We are open to pull-requests with proposals on how to implement multicast for our Dockerfile.
```
Comment (rbotzer) [inline README.md:140]: Removing the mention even because there will be no multicast support

File: README.md [hunk 4]
```diff
@@ -149,19 +191,12 @@ Mesh networking requires setting up links between each node in the cluster. This
```
Comment (rbotzer) [inline README.md:191]: @volmarl Is the tip command recommendation still valid?

File: README.md (hunk 2 above)
Comment (volmarl) [reply README.md:138]: It could be any local folder <Local Absolute path> where docker run is running.

File: README.md (hunk 4 above)
Comment (volmarl) [reply README.md:191]: Its still valid for people building a cluster from docker run or with dynamic ip addresses.

**Comment by rbotzer** [comment]
Moving this to citrusleaf

**Pull request closed without merging**